monitoring/uss_qualifier/scd/simulator/utils.py

```diff
@@ -0,0 +1,104 @@
+    
+class GeneratedGeometry(ImplicitDict):
+    ''' An object to hold generated flight path, is_control is a nomenclature used to see if the generated path is the first one (against which the geometry and vole generation rules are applied) '''
+    geometry: Union[LineString, Polygon]
+    is_control: bool
+    geometry_generation_rule: GeometryGenerationRule
+  
+class GeneratedFlightPlan(ImplicitDict):
+    ''' A class used internally in the simulator with altiude and time details and GeoJSON as a flight plan the GeoJSON flight plan is converted in to a Volume 4D object and written to disk subsequently '''
+    flight_plan: Feature 
+    time_start: StringBasedDateTime
+    time_end: StringBasedDateTime
+    altitude_lower: Altitude
+    altitude_upper: Altitude
+    
+class OperationalIntentInjection(ImplicitDict):
+    ''' A class to hold data for operational intent data that will be submitted to the SCD testing interface.  For more information see : https://github.com/interuss/dss/blob/master/interfaces/automated-testing/scd/scd.yaml#L286'''
+    state: Literal[OperationalIntentState.Accepted,OperationalIntentState.Activated,OperationalIntentState.Nonconforming, OperationalIntentState.Contingent] 
+    priorty: int = 0
+    volumes: List[Volume4D]
+    off_nominal_volumes: List[Volume4D]= []
+
+
+## Definitions around flight authorization data that needs to be submitted to the test injection interface
+    
+class OperationMode(str, enum.Enum):
+    ''' A enum to hold all modes for an operation, for more information see: https://github.com/interuss/dss/blob/master/interfaces/automated-testing/scd/scd.yaml#L393 '''
+    Undeclared = 'Undeclared'
+    Vlos = 'Vlos'
+    Bvlos = 'Bvlos'
+
+class OperationCategory(str, enum.Enum):
+    ''' A enum to hold all categories for an operation '''
+    Open = 'Open'
+    Specific = 'Specific'
+    Certified = 'Certified'
+
+class UASClass(str, enum.Enum):
+    ''' A object to hold all UAS Classes per EASA defintions'''
+    Other = 'Other'
+    C0 = 'C0'
+    C1 = 'C1'
+    C2 = 'C2'
+    C3 = 'C3'
+    C4 = 'C4'
+    C5 = 'C5'
+    C6 = 'C6'
+
+class IDTechnology(str, enum.Enum):
+    ''' A enum to hold ID technologies for an operation '''
+    Network = 'network'
+    Broadcast = 'broadcast'
+
+class FlightAuthorizationData(ImplicitDict):
+    '''A class to hold information about Flight Authorization Test, for more information see https://github.com/interuss/dss/blob/master/interfaces/automated-testing/scd/scd.yaml#L317'''
+    
+    uas_serial_number: str
+    operation_mode: Literal[OperationMode.Undeclared, OperationMode.Vlos, OperationMode.Bvlos]
+    operation_category: Literal[OperationCategory.Open, OperationCategory.Specific, OperationCategory.Certified]
+    uas_class: Literal[UASClass.Other,UASClass.C0, UASClass.C1, UASClass.C2, UASClass.C3, UASClass.C4,UASClass.C5, UASClass.C6]
+    identification_technologies: List[str]
```

Comment:
We have the `IDTechnology` type defined above; we should use it here in typing or delete it.

Reply:
I have now removed `IDTechnology`